src/console_reporter.cc

```diff
@@ -92,7 +95,9 @@ void ConsoleReporter::PrintRunData(const Run& result) {
                    " items/s");
   }
 
-  double const multiplier = 1e9; // nano second multiplier
+  double const multiplier = time_unit_ == "ns" ? 1e9 : 1e3; // nano second or
```

Comment:
we take pains to make other values human readable. could we do the same here?

ie, instead of a flag, can we add the unit suffix to the output and automatically detect if the value should be ns, us, ms, etc?